### ISSUE TITLE
disallowedClasses does not work when nullable

### ISSUE THREAD

**Issue by javaDeveloperKid** [spaze/phpstan-disallowed-calls#131]
Hello, I found out that nullable types are not reported by this extension. 
```php
class MyClass
{
    public \DateTime $datetime; // is reported
    public ?\DateTime $datetime2; // does not report
}
```

**Comment by spaze** [comment]
Hi, thanks for the report. You're right, nullable types are not supported yet. I have a fix ready (also for union and intersection types), just need to figure out how to skip the new tests on older PHPs without union/intersection support. I'll let you know here once the fix is released.

**Comment by spaze** [comment]
Just released the fix in [2.8.0](https://github.com/spaze/phpstan-disallowed-calls/releases/tag/v2.8.0).
